.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "✨Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem?
```

Comment:
why describe it as a question? Is this a check for them?
Why not "## Feature description"

Reply:
Yeah, I borrowed github's default template here... I thought this was a little weird as well. I'm +1 for `Describe the feature in detail and any related problems it solves`